### PR TITLE
Sys access everything

### PR DESCRIPTION
**Pull request by jworcestBAH**
Allowing users of the 'SYS' agency to view all submissions

### CODE REVIEW

File: dataactbroker/handlers/fileHandler.py [hunk 1]
```diff
@@ -195,9 +195,12 @@ def checkSubmissionPermission(self,submission):
         """
         userId = LoginSession.getName(session)
         user = self.interfaces.userDb.getUserByUID(userId)
-        # Check that user has permission to see this submission, user must be within the agency of the submission
-        if(submission.cgac_code != user.cgac_code and submission.user_id != user.user_id):
-            raise ResponseException("User does not have permission to view that submission",StatusCode.CLIENT_ERROR)
+        # Check that user has permission to see this submission, user must be within the agency of the submission, or be
+        # the original user, or be in the 'SYS' agency
+        if(submission.cgac_code != user.cgac_code and submission.user_id != user.user_id
+           and user.cgac_code != "SYS"):
```
Comment (nmonga91) [inline dataactbroker/handlers/fileHandler.py]: I didn't think of this when converting to cgac_code but should those comparisons be case sensitive?

Comment (jworcestBAH) [inline dataactbroker/handlers/fileHandler.py]: Good catch, fixed that and stripping whitespace

**Comment by jworcestBAH** [comment]
Fixed to use cleanString method which is now in the core.

**Comment by jworcestBAH** [comment]
Sigh it passed unit tests locally.

**Comment by jworcestBAH** [comment]
Jenkins: Test Result (no tests)

**Comment by jworcestBAH** [comment]
Try harder Jenkins [run tests]

**Comment by nmonga91** [comment]
Looks good! Merging

**Pull request merged**